tests/conftest.py

```diff
@@ -106,6 +107,48 @@ def bootstrap_config(host):
         return yaml.safe_load(config_file.content_string)
 
 
+@pytest.fixture
+def control_plane_ip(host):
+    with host.sudo():
+        ip_json = host.check_output(
+            "salt-call grains.get metalk8s:control_plane_ip --out json"
+        )
+
+    return json.loads(ip_json)["local"]
+
+
+@pytest.fixture
+def build_image_name(k8s_client, control_plane_ip, version):
```

Comment:
Does this need to be a fixture or can it be simply an helper?

If we want a fixture, maybe rename it `image_name_builder`.
Usually, fixture name behave like "variable" instead of "function" (i.e.: they're not named like action => `control_plane_ip` vs `get_control_plane_ip`).

Reply:
I wanted a fixture for easy access, since I don't see other use cases for a `registry_address` fixture. The thing is, fixtures are named as the variables that would hold their return value. In that particular case, the return value of the fixture **is** a function, so I find it more explicit than the abstract `_builder` suffix.